web/src/styles/app/form.css

```diff
@@ -33,3 +33,33 @@
 .select-condensed {
   height: 2rem;
 }
+
+.rdw-editor-main .DraftEditor-root {
+  min-height: unset; /* override of plugin */
+}
+
+/* add resizing capability to divs of react-draft-wysiwyg plugin */
+.public-DraftEditor-content {
+  resize: vertical;
+}
+
+/* rte-* classes are for react-draft-wysiwyg plugin */
```

Comment:
Because the RTE plugin only allows you to specify a class for one of the parent divs that wraps the actual "textarea" div, I had to target the RTE "textareas" slightly more ham-handedly than regular `textarea`s.